Kentico.Kontent.Management.Tests/Unit/CodeSamples/CmApiV2.cs

```diff
@@ -612,18 +653,18 @@ public async void PatchTaxonomyGroup()
             {
                 new TaxonomyGroupReplacePatchModel
                 {
-                    PropertyName = PropertyName.Name,
+                    PropertyName = Models.TaxonomyGroups.Patch.PropertyName.Name,
```

Comment:
It's sad that it needs to be specified with the full name, but I guess in real-world application user would use aliases.

Reply:
Yeap - I didn't want to use aliases for tests. But in the app I would.